### PR TITLE
Improvement: change ui of pagination from load more to next-previous  for services page

### LINKED ISSUE #1277: UI - Service Page needs pagination - shows only 10 services
> The [services page](http://localhost:8585/services) shows maximum 10 services on UI, need to add pagination.

image
> The [service tab on explore page](http://localhost:8585/my-data) shows 10 services

image
> The network tab shows total of 11 services.

image

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
Closes #1277 

### Describe your changes :

Change pagination UI for services page from load more pagination to next-previous pagination.

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

 Frontend: @shahsank3t, @darth-coder00, @Sachin-chaurasiya

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx [hunk 1]
```diff
@@ -659,16 +646,14 @@ const ServicesPage = () => {
...
-              <div className="tw-my-4 tw-flex tw-justify-center tw-items-center">
-                <LoadMorePagination
-                  showLoadingText
-                  buttonText="Load more"
-                  handleClick={() => pagingHandler(CursorType.AFTER)}
-                  isLoading={isLoadingMore}
-                  status={status}
-                />
-              </div>
+            {Boolean(
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx:649]: @ShaileshParmar11 isNil will return boolean so no need to cast it with `Boolean`.

**Review by Sachin-chaurasiya — APPROVED** [review]
One minor comment otherwise, LGTM.

File: openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx (hunk 1 above)
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx:649]: @ShaileshParmar11 isNil will return boolean so no need to cast it with Boolean.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
One minor change, otherwise LGTM.

File: openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx]: updated.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**